packages/format-library/src/link/modal-screens/link-settings-screen.native.js

```diff
@@ -213,6 +217,15 @@ const LinkSettingsScreen = ( {
 							onValueChange={ setOpensInNewWindows }
 							separatorType={ 'fullWidth' }
 						/>
+						<BottomSheet.SwitchCell
+							icon={ external }
+							label={ __(
+								'Search engines should ignore this link (mark as nofollow)'
+							) }
+							value={ markedAsNoFollow }
```

Comment:
Since we are using the `markedAsNoFollow` value here and it's within a `useMemo` we should include it in the dependencies so it gets the updated value.